### DISCUSSION TITLE
DB query is getting lower case

### DISCUSSION THREAD

**Discussion by eamnicoletti** [serverpod/serverpod#709] [Q&A] [answered]
When I use:
`session.db.query() or AuthKey.update()`

I got this error:
`2023-03-05 23:49:13.839150Z Internal server error. Zoned exception.
PostgreSQLSeverity.error 42703: column "userid" does not exist Hint: Perhaps you meant to reference the column "serverpod_auth_key.userId". `

I'm using the right column name, but looks like it's turning my query string to lower case.

**Comment by vlidholt** [comment]
I think you need to put the column names in quotes or they will be made lower case.

**Reply by eamnicoletti** [reply]
I've tried that too. But guess what, I just restarted the machine today and it worked like magic. Anyway, thanks for following-up my question.

**Comment by eamnicoletti** [comment, marked as answer]
At that time, just to restart the machine works for me.

**Comment by AvoMandjian** [comment]
I have also faced this issue, even when you do the query inside a postgresql GUI it takes lower case, i have changed column names to Snake case but it's not recommended by flutter. I'll try restarting the machine next time and see if it'll work, thanks for the suggestion

**Reply by eamnicoletti** [reply]
I don't know if they are directly related, but in my case the issue appeared after I had an exception with quotes in a text field, which I soon noticed and corrected, but then I started having problems with lowercase letters. Like it's a cache problem.
